### DISCUSSION TITLE
Stepper styles and passthrough improvements

### DISCUSSION THREAD

**Discussion by sandrosimas** [primefaces/community#2346] [PrimeReact]
- If we set a background color different than white on a page, this is how the stepper header looks. There's a white background on the button. Instead of doing this, could you not set a background color, or define a padding for the button?
Current style:
<img width="844" alt="image" src="https://github.com/user-attachments/assets/e8edcc1d-0c08-4adc-b0d1-4bfb7825ffa7">

Without background color:
<img width="842" alt="image" src="https://github.com/user-attachments/assets/fd9273db-1ce1-4fb5-91b5-121c3b389304">
With padding:
<img width="865" alt="image" src="https://github.com/user-attachments/assets/dba6d803-df35-4b06-8301-4c145ba8c646">

- The panel has no border-radius. It could be like a card;

Current style:
<img width="728" alt="image" src="https://github.com/user-attachments/assets/193a70f5-fe92-44c0-90d7-72f29e611566">
With card border-radius:
<img width="781" alt="image" src="https://github.com/user-attachments/assets/55b9878b-beb7-4836-9fe1-c59b87618acc">

- There's no way to customize stepper header and action through passthrough. These are all the options:
<img width="380" alt="image" src="https://github.com/user-attachments/assets/8d553aed-7381-4792-b79a-a2da07b26d98">

- There's no way to customize stepper globally:
<img width="889" alt="image" src="https://github.com/user-attachments/assets/77f3500f-2cea-4372-8307-b7717d0cc28f">

**Comment by melloware** [comment]
PR is welcome
